sdk/py.typed

```diff
@@ -0,0 +1,27 @@
+# --------------------------------------------------------------------------
```

Comment:
This file shouldn't be here, it's supposed to be package at each individual level, not outside of a package